From 99c265bd8b922b48d38f948a71b98954f8902567 Mon Sep 17 00:00:00 2001
From: Vagrant Cascadian <vagrant@debian.org>
Date: Mon, 29 Aug 2016 00:30:00 -0700
Subject: [PATCH 1/4] omap3_pandora: Fix mmc loading of autoboot script to use
 correct syntax.

fatload/ext2load both require that the device and partition be
specified after specifying the device type. Specify the first
partition on mmc device 0, which is the only mmc device currently
configured on the pandora.
---
 include/configs/omap3_pandora.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/configs/omap3_pandora.h b/include/configs/omap3_pandora.h
index 380ec12..6ff9375 100644
--- a/include/configs/omap3_pandora.h
+++ b/include/configs/omap3_pandora.h
@@ -93,8 +93,8 @@
 	"mtdparts=" MTDPARTS_DEFAULT "\0" \
 
 #define CONFIG_BOOTCOMMAND \
-	"if mmc rescan && fatload mmc1 0 ${loadaddr} autoboot.scr || " \
-			"ext2load mmc1 0 ${loadaddr} autoboot.scr; then " \
+	"if mmc rescan && fatload mmc 0:1 ${loadaddr} autoboot.scr || " \
+			"ext2load mmc 0:1 ${loadaddr} autoboot.scr; then " \
 		"source ${loadaddr}; " \
 	"fi; " \
 	"ubi part boot && ubifsmount ubi:boot && " \
-- 
2.9.3

